OneSignalSDK/onesignal/notifications/build.gradle

```diff
@@ -77,7 +77,7 @@ dependencies {
     // firebase-messaging:23.0.0 incoporates fix for SecurityException: Not allowed to bind to service
```

Comment:
We have some comments above that are no longer relevant /  we can add a comment about why moving to version 21...
```
    // firebase-messaging:18.0.0 is the last version before going to AndroidX
    // firebase-messaging:19.0.0 is the first version using AndroidX
```

Reply:
Good point! I have pushed the change for the comment.